### PR TITLE
remove authorized_key; use ansible builtins

### PR DESCRIPTION
**Pull request by richm**
remove authorized_key; use ansible builtins

### CODE REVIEW

File: tasks/ssh.yml [hunk 1]
```diff
@@ -12,13 +12,51 @@
...
+- name: Write new authorized_keys if needed
+  vars:
+    __kdump_authorized_keys_lines: "{{
+      (__kdump_authorized_keys.content | b64decode).split('\n') |
+      reject('match', '^$') | list
+      if __kdump_authorized_keys is defined else []
+    }}"
+    __kdump_authorized_keys_path: "{{
+      __kdump_authorized_keys_file.stat.path |
+      d(__kdump_ssh_user_info.home ~ '/.ssh/authorized_keys')
```
Comment (spetrosi) [inline tasks/ssh.yml:46]: In a task above, `"{{ __kdump_ssh_user_info.home ~ '/.ssh/authorized_keys' }}"` is registered into `__kdump_authorized_keys_file`. Why do you provide the default here?

Comment (richm) [reply tasks/ssh.yml:46]: > In a task above, `"{{ __kdump_ssh_user_info.home ~ '/.ssh/authorized_keys' }}"` is registered into `__kdump_authorized_keys_file`.

Only if the file exists.  If the file doesn't exist, then `__kdump_authorized_keys_file.stat.path` will be undefined.

> Why do you provide the default value with `d(..)` here?

**Comment by richm** [comment]
@spetrosi how about this instead?  https://github.com/linux-system-roles/kdump/pull/88/commits/87f965fd4828fa04b272b583ea7a0c53d35d4ee1

**Review by spetrosi — APPROVED** [review]
lgtm

**Pull request merged**